test/remoteIncludes.spec.js

```diff
@@ -11,10 +11,10 @@ let harvester = require('../lib/harvester');
 
 let harvesterOptions = {
   adapter: 'mongodb',
-  connectionString: 'mongodb://127.0.0.1:27017/testDB',
+  connectionString: process.env.MONGODB_URL || 'mongodb://db:27017/testDB',
```

Comment:
I'm not familiar with db as shorthand - does that mean localhost - and will that play well with non-dockerized users?